src/source_info.rs

```diff
@@ -809,13 +809,19 @@ impl<'i> SourceInfoBuilder<'i> {
     }
 
     fn find_module(&self, name: &str) -> Option<Arc<Url>> {
-        let Ok(path) = koto_bytecode::find_module(name, None) else {
-            return None;
-        };
```

Comment:
`find_module` previously took `None`, so I think you could pass the result of `parent()` to avoid the `unwrap` here. Or bail is `parent()` is `None`.

Reply:
Uuups, sorry, will fix that later